### DISCUSSION TITLE
High latency between saved filename and image timestamp

### DISCUSSION THREAD

**Discussion by CRCinAU** [Motion-Project/motionplus#81] [Q&A] [answered]
### Did you read the guide?

Yes

### What is the base version number of MotionPlus being used?

0.1.x

### What was the install method?

Built from source code

### What is base architecture?

ARM-64bit

### What is the distro being used?

Ubuntu

### Camera/Sound type(s) being used?

Network camera with RTSP

### Describe the issue/problem and steps to reproduce

When saving snapshots using the filename structure `%Y%m%d%H%M%S-%q`, an image is saved correctly, but the cameras timestamp is about 6 seconds delayed.

ie a filename of 20230711172724-00.jpg has the visible timestamp from the netcam of 17:27:18.

When using the same setup with Motion, the difference seemed to consistently be 1 second.

There doesn't seem to be any options for buffering etc that would make this any different.

### Relevant MotionPlus log output (at log_level 8)

```shell
N/A
```

**Comment by Mr-Dave** [comment, marked as answer]
I've solved this before by specifying a larger capture_rate.

**Reply by CRCinAU** [reply]
Ok, right now I was using:
```
netcam_params                capture_rate=pts
netcam_high_params      capture_rate=pts
```

Given the snapshots are taken from the `netcam_high_url`, should I change pts to something higher?

Ah - yes. Changing to the following went back to 1s delay:
```
netcam_high_params      capture_rate=30
```
